### PR TITLE
Remove debugging from CI

### PR DESCRIPTION
**Pull request by szepeviktor**
👋🏻 Debugging! 👋🏻 

And some improvements to make shell commands safer.

### CODE REVIEW

**Comment by szepeviktor** [comment]
Strict quoting revealed something... 🔎

File: bin/acceptance-tests.sh [hunk 1]
```diff
@@ -8,48 +8,50 @@ echo "Starting up..."
...
-CHROME_PORT=`docker inspect --type=container --format='{{(index .NetworkSettings.Ports "4444/tcp" 0).HostPort}}' qm-chrome`
-DATABASE_PORT=`docker inspect --type=container --format='{{(index .NetworkSettings.Ports "3306/tcp" 0).HostPort}}' qm-database`
+WP_PORT="$(docker inspect --type=container --format='{{(index .NetworkSettings.Ports "80/tcp" 0).HostPort}}' qm-server)"
+CHROME_PORT="$(docker inspect --type=container --format='{{(index .NetworkSettings.Ports "4444/tcp" 0).HostPort}}' qm-chrome)"
+DATABASE_PORT="$(docker inspect --type=container --format='{{(index .NetworkSettings.Ports "3306/tcp" 0).HostPort}}' qm-database)"
 WP_URL="http://host.docker.internal:${WP_PORT}"
-WP="docker-compose run --rm wpcli --url=${WP_URL}"
+wp() {
+	docker-compose run --rm wpcli --url="${WP_URL}" "$@"
+}
```
Comment (szepeviktor) [inline bin/acceptance-tests.sh:17]: Executing _remote_ WP-CLI could be implemented in a shell function.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -44,8 +44,8 @@ jobs:
       run: npm run build
 
     - name: Setup
-      run: >
-        echo VERSION=$(node -p "require('./package.json').version") >> $GITHUB_ENV
+      run: |
+        echo "VERSION=$(node -p "require('./package.json').version")" >> $GITHUB_ENV
```
Comment (johnbillion) [inline .github/workflows/build.yml:48]: Does this still work with the double quotes inside double quotes?

Comment (szepeviktor) [reply .github/workflows/build.yml:48]: In YAML Specs `>` means to fold line ends,
`|` means to keep line ends.

Comment (szepeviktor) [reply .github/workflows/build.yml:48]: > Does this still work with the double quotes inside double quotes?

I see! The inner quotes are in `$(   )` That works in Bash.

**Comment by szepeviktor** [comment]
Resolving conflicts ...
🏃🏻‍♂️

**Comment by szepeviktor** [comment]
People says: Now it works.
I say: Now you don't see the error! 🙃

**Comment by szepeviktor** [comment]
I like the minimal workflow configs in this repository!

**Pull request closed without merging**